cdap-app-fabric/src/main/java/co/cask/cdap/scheduler/CoreSchedulerService.java

```diff
@@ -212,6 +213,16 @@ public void updateSchedule(final ProgramSchedule schedule) throws NotFoundExcept
       throw new IllegalStateException(
         "Schedule '" + schedule.getScheduleId() + "' already exists despite just being deleted.");
     }
+    // if the schedule was previously enabled, it should still/again enabled be after the update
+    if (ProgramScheduleStatus.SCHEDULED == previousStatus) {
+      try {
+        enableSchedule(schedule.getScheduleId());
```

Comment:
the documented behavior is that schedules are not enabled when they are created, and they must be enabled explicitly. update(), however, did preserve the status of the schedule in previous code - now that we delete and re-add, we need to preserve that behavior. Our test cases actually rely on this.